adbacli.py

```diff
@@ -115,13 +113,18 @@
         connection.auth(args.username, args.password)
     except Exception as e:
         print('Exception: %s', e)
+        logging.debug('Exception Occurred: ' + str(e))
```

Comment:
I would use `%r` and have the occurred lowercase. Same for all the others.

Reply:
Also the line above does not print as you would expect.
```python
>>> e = 'test'
>>> print('Exception: %s', e)
Exception: %s test
```